Ert unit_tests/gui/test_main_window.py::test_that_gui_plotter_works_when_no_data fails and seems flaky

**Describe the bug**
The unit test often fails with 
```
FAILED unit_tests/gui/test_main_window.py::test_that_gui_plotter_works_when_no_data - assert 5 == 0
 +  where 5 = len([<ert.gui.tools.plot.plot_case_selection_widget.CaseSelectCheckButton object at 0x7f2a819841f0>, <ert.gui.tools.plot.plot_case_selection_widget.CaseSelectCheckButton object at 0x7f2a81984280>, <ert.gui.tools.plot.plot_case_selection_widget.CaseSelectCheckButton object at 0x7f2a81984310>, <ert.gui.tools.plot.plot_case_selection_widget.CaseSelectCheckButton object at 0x7f2a819843a0>, <ert.gui.tools.plot.plot_case_selection_widget.CaseSelectCheckButton object at 0x7f2a81984430>])
```
The test creates a minimal ert config, and attempts plotting it before it is even ran. It should not have any data to plot, but apparently it finds it somewhere. Could it be that the storage fixture is not working correctly, and multiple tests are sharing data?
The test is very short and not complicated, which should help troubleshooting a lot.

**To reproduce**
Steps to reproduce the behaviour:
1. Run nightly build, or we might be able to reproduce it by running on an RGS node. If not, try running it with pytest-repeat.
2. Fails while running with `pytest -x tests/unit_tests/gui/test_main_window.py -k plot -n 12 --count 50`

**Expected behaviour**
Test should not fail

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - OS: RHEL7
 - ERT/Komodo release: Bleeding
 - Python version: 3.8
 - Remote/HPC execution involved: Azure

**Additional context**
Add any other context about the problem here.

--- a/tests/integration_tests/scheduler/test_integration_local_driver.py
+++ b/tests/integration_tests/scheduler/test_integration_local_driver.py
@@ -45,12 +45,13 @@ async def test_subprocesses_live_on_after_ert_dies(tmp_path, try_queue_and_sched
     create_ert_config(tmp_path)
 
     ert_process = subprocess.Popen(["ert", "test_run", "ert_config.ert"], cwd=tmp_path)
-    check_path_retry, check_path_max_retries = 0, 50
+    check_path_retry, check_path_max_retries = 0, 60
     expected_path = Path(tmp_path, "test_out/realization-0/iter-0/forward_model_pid")
     while not expected_path.exists() and check_path_retry < check_path_max_retries:
         check_path_retry += 1
-        await asyncio.sleep(0.5)
+        await asyncio.sleep(1)
 
+    assert ert_process.poll() is None
     assert expected_path.exists()
     child_process_id = expected_path.read_text(encoding="utf-8").strip()
     assert child_process_id

--- a/tests/unit_tests/gui/simulation/test_run_dialog.py
+++ b/tests/unit_tests/gui/simulation/test_run_dialog.py
@@ -380,7 +380,7 @@ def test_that_run_dialog_can_be_closed_while_file_plot_is_open(
         qtbot.mouseClick(run_dialog.show_details_button, Qt.LeftButton)
         job_view = run_dialog._job_view
         qtbot.waitUntil(job_view.isVisible, timeout=20000)
-        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=20000)
+        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=200000)
 
         realization_widget = run_dialog.findChild(RealizationWidget)
 

--- a/tests/unit_tests/gui/test_main_window.py
+++ b/tests/unit_tests/gui/test_main_window.py
@@ -40,7 +40,7 @@ from ert.gui.tools.plot.plot_case_selection_widget import (
     CaseSelectCheckButton,
     CaseSelectionWidget,
 )
-from ert.gui.tools.plot.plot_window import PlotWindow
+from ert.gui.tools.plot.plot_window import PlotApi, PlotWindow
 from ert.run_models import SingleTestRun
 from ert.services import StorageService
 from ert.shared.plugins.plugin_manager import ErtPluginManager
@@ -765,13 +765,13 @@ def test_that_a_failing_job_shows_error_message_with_context(
 
 
 @pytest.mark.usefixtures("use_tmpdir")
-def test_that_gui_plotter_works_when_no_data(qtbot, storage):
+def test_that_gui_plotter_works_when_no_data(qtbot, storage, monkeypatch):
+    monkeypatch.setattr(PlotApi, "_get_all_cases", lambda _: [])
     config_file = "minimal_config.ert"
     with open(config_file, "w", encoding="utf-8") as f:
         f.write("NUM_REALIZATIONS 1")
     args_mock = Mock()
     args_mock.config = config_file
-
     ert_config = ErtConfig.from_file(config_file)
     enkf_main = EnKFMain(ert_config)
     with StorageService.init_service(
@@ -782,7 +782,6 @@ def test_that_gui_plotter_works_when_no_data(qtbot, storage):
         gui.notifier.set_storage(storage)
         qtbot.addWidget(gui)
         gui.tools["Create plot"].trigger()
-
         plot_window = wait_for_child(gui, qtbot, PlotWindow)
         case_selection = get_child(plot_window, CaseSelectionWidget)
         assert isinstance(case_selection, CaseSelectionWidget)

--- a/tests/unit_tests/gui/test_main_window.py
+++ b/tests/unit_tests/gui/test_main_window.py
@@ -279,7 +279,7 @@ def test_that_run_dialog_can_be_closed_after_used_to_open_plots(
         assert not run_dialog.isModal()
 
         qtbot.mouseClick(run_dialog.plot_button, Qt.LeftButton)
-        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=20000)
+        qtbot.waitUntil(run_dialog.done_button.isVisible, timeout=200000)
         qtbot.mouseClick(run_dialog.done_button, Qt.LeftButton)
 
         # Ensure that once the run dialog is closed

